pkgs/by-name/bo/bootcommander/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libopenblt,
+}:
+
+let
+  pname = "bootcommander";
+  version = "1.15.00";
+in
+stdenv.mkDerivation rec {
+
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "feaser";
+    repo = "openblt";
+    rev = "openblt_v0${lib.replaceStrings [ "." ] [ "" ] version}"; # will break at version 10.x.x
```

Comment:
```suggestion
    rev = "refs/tags/openblt_v0${lib.replaceStrings [ "." ] [ "" ] finalAttrs.version}"; # will break at version 10.x.x
```

Reply:
why? its working fine without that?

Reply:
To disambiguate from branches and other refs